pkgs/development/tools/gertty/default.nix

```diff
@@ -0,0 +1,48 @@
+{ python3, python3Packages, stdenv }:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "gertty";
+  version = "1.6.0";
+
+  nativeBuildInputs = with python3Packages; [
+    pbr
+    voluptuous
+    dateutil
+    requests
+    urwid
+    ply
+    ordereddict
+    alembic
+    GitPython
+    pyyaml
+  ];
+
+  # XXX Looks like all the buildInputs needs to be propagated...
```

Comment:
To why see: https://nixos.org/nixpkgs/manual/#reference 15.17.2.2.1.1. buildPythonPackage parameters

Reply:
Thanks, I had quite a read :)